CardsCashCasino/BlackjackManager.cs

```diff
@@ -0,0 +1,215 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection;
+
+namespace CardsCashCasino.Game
+{
+    public class BlackjackManager
+    {
+        private CardManager _cardManager; // Instance of CardManager for deck management
+        private List<Card> _dealerHand; // Dealer's hand
+        private List<Card> _playerHand; // Player's hand
+        private List<Card> _playerSplitHand; // Player's split hand
+        private bool _gameOver; //if player busts, allows game to skip dealer action
+        private bool _splitHand; //activates split hand functionality if player splits
+
+        public BlackjackManager()
+        {
+            _cardManager = new CardManager();
+            _dealerHand = new List<Card>();
+            _playerHand = new List<Card>();
+            _playerSplitHand = new List<Card>();
+            _gameOver = false;
+            _splitHand = false;
+        }
+
+        /// <summary>
+        /// Starts a new game by generating and shuffling the deck, then dealing initial cards.
+        /// </summary>
+        public void StartNewGame()
+        {
+            _playerHand.Clear();
+            _playerSplitHand.Clear();
+            _dealerHand.Clear();
+            _gameOver = false;
+            _splitHand = false;
+            _cardManager.ClearDecks();
+            _cardManager.GenerateDecks(1);
+            _playerHand.Add(_cardManager.DrawCard());
+            _dealerHand.Add(_cardManager.DrawCard());
+            _playerHand.Add(_cardManager.DrawCard());
+            _dealerHand.Add(_cardManager.DrawCard());
+        }
+
+        public int GetBlackjackValue(List<Card> hand)
+        {
+            int value = 0;
+            int aceCount = 0;
+
+            foreach (Card card in hand)
+            {
+                switch (card.Value)
+                {
+                    case Value.Two:
+                    case Value.Three:
+                    case Value.Four:
+                    case Value.Five:
+                    case Value.Six:
+                    case Value.Seven:
+                    case Value.Eight:
+                    case Value.Nine:
+                    case Value.Ten:
+                        value += (int)card.Value; // Number cards are worth their face value
+                        break;
+                    case Value.Jack:
+                    case Value.Queen:
+                    case Value.King:
+                        value += 10; // Face cards are worth 10 points each
```

Comment:
There's a way to get the BlackjackValue of these cards from that attribute of the face card Enum objects that we should use. I'll approve this PR if Ethan includes this in his minor edits of this code.